android/build.gradle

```diff
@@ -127,4 +131,5 @@ dependencies {
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation 'androidx.viewpager2:viewpager2:1.0.0'
+  implementation "androidx.core:core-ktx:1.6.0"
```

Comment:
Could you explain, why core-ktx has been added ?

Reply:
Because I used view.doOnLayout {} and it is only available when using core-ktx extension. You can see further information in this doc below https://developer.android.com/reference/kotlin/androidx/core/view/package-summary#(android.view.View).doOnLayout(kotlin.Function1)